src/styles/styles.less

```diff
@@ -580,6 +580,7 @@ hr {
     }
 
     &.canceled {
+      text-decoration: line-through;
```

Comment:
in terms of HTML semantics, `<del>` (or maybe `<s>`) is probably a better bet here, and it would usually provide strikeout. some of its implications discussed here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del

rather than resorting to inserting pseudo-elements with text as described in that article, though, i reckon an informative `aria-label` or `aria-description` (note that you get the latter for free when you use v10's `data-tooltip` attribute) would do the trick.

Reply:
I just pushed a thing that wraps the die span in a `<del>`, like so:

```html
<del>
  <span class="action roll d6"> ... </span>
</del>
```

Does that satisfy? It doesn't look like it gets a `line-through` by default, and the orange highlight would have been missing anyways, so I still needed to add a style blurb.